### PR TITLE
feat(ui5-expandable-text): add ExpandableText component

### PR DESCRIPTION
**Pull request by dimovpetar**
BGSOFUIRODOPI-3367

### CODE REVIEW

File: packages/main/src/ExpandableText.ts [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+ * @class
+ *
+ * ### Overview
+ *
+ * The `ui5-expandable-text` component allows displaying a large body of text in a small space. It provides an "expand/collapse" functionality, which shows/hides potentially truncated text.
+ *
+ * ### Usage
+ *
+ * #### When to use:
+ * - To accommodate long texts in limited space, for example in list items or table cell texts
```
Comment (TeodorTaushanov) [inline packages/main/src/ExpandableText.ts:38]: To accommodate long texts in limited space, for example in list items or table cell texts **or forms**

**Review by TeodorTaushanov — CHANGES_REQUESTED** [review]
- Add aria-labelledby of the Popover that points to the text.
- Increase the height of the public samples

File: packages/main/src/ExpandableText.ts (hunk 1 above)
Comment (dimovpetar) [reply packages/main/src/ExpandableText.ts:38]: Done

**Comment by dimovpetar** [comment]
> * Add aria-labelledby of the Popover that points to the text.
> * Increase the height of the public samples

Done

**Review by vladitasev — COMMENTED** [review]
Looks good in terms of API - no slots, just properties

File: packages/main/src/ExpandableText.ts [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+ * - The content is critical for the user. In this case use short descriptions that can fit in
+ * - Strive to provide short and meaningful texts to avoid excessive number of "Show More" links on the page
+ *
+ * ### Responsive Behavior
+ *
+ * On phones, when the component is set to show the full text in a popover, the popover will be displayed full screen.
+ *
+ * ### ES6 Module Import
+ *
+ * `import "@ui5/webcomponents/dist/Text";`
```
Comment (TeodorTaushanov) [inline packages/main/src/ExpandableText.ts:50]: ExpandableText

**Review by TeodorTaushanov — CHANGES_REQUESTED** [review]

File: packages/main/src/i18n/messagebundle.properties [hunk 3]
```diff
@@ -170,6 +170,21 @@ EMPTY_INDICATOR_SYMBOL=\u2013
 #XFLD: ARIA announcement for the empty value.
 EMPTY_INDICATOR_ACCESSIBLE_TEXT=Empty value
 
+#XLNK: Link to allow the user to see complete text
+EXPANDABLE_TEXT_SHOW_MORE=Show More
+
+#XLNK: This link allows the user to collapse the complete text and display only the first characters that can fit
```
[the remark this answers is not in the document]
Comment (dimovpetar) [reply packages/main/src/i18n/messagebundle.properties:176]: Done

**Review by TeodorTaushanov — APPROVED** [review]

**Review by Stoev — COMMENTED** [review]
UA all good.

**Pull request merged**